### PR TITLE
Added some checks for proxy

### PR DESCRIPTION
**Pull request by Anumita**
---

I haven't updated the version in setup.py as akash will be updating in a later PR this week

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedk8s

**Review by akashkeshari — APPROVED** [review]

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 1]
```diff
@@ -590,15 +592,18 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, resourc
...
-                        "--set", "global.httpsProxy={}".format(https_proxy),
-                        "--set", "global.httpProxy={}".format(http_proxy),
-                        "--set", "global.noProxy={}".format(no_proxy),
                         "--set", "global.onboardingPrivateKey={}".format(private_key_pem),
                         "--set", "systemDefaultValues.spnOnboarding=false",
                         "--output", "json"]
     # To set some other helm parameters through file
     if values_file_provided:
         cmd_helm_install.extend(["-f", values_file])
+    if https_proxy != "":
```
Comment (fengzhou-msft) [inline src/connectedk8s/azext_connectedk8s/custom.py:601]: Suggest to use `if https_proxy:`

Comment (Anumita) [reply src/connectedk8s/azext_connectedk8s/custom.py]: done

**Comment by Anumita** [comment]
@fengzhou-msft , review is done from the connectedk8s side as well. It is ready for checking in if you don't have any further reviews

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**